### PR TITLE
More detailed explanatory text for `BACKEND` of README file

### PR DESCRIPTION
**Pull request by yillkid**
More detailed explanatory text for `BACKEND`config including usage and how to deploy

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -53,6 +53,8 @@ device is lost.
 
 - [x] Create new file named `.env` and specify `BACKEND` and `API_HOST`
   * `BACKEND`: an address (IP or domain) providing a proxy to bypass CORS.
+    * `BACKEND` is a open source extension named [iota-swarm-node](https://github.com/yillkid/iota-swarm-node) be used with IOTA's IRI Node. It format TangleID requests meet [DID Specification](https://w3c-ccg.github.io/did-spec/).
```
Comment (jserv) [inline README.md]: Fix syntax error: "an" open source.

**Comment by jserv** [comment]
Ask Spin for syntax checking.

**Comment by jserv** [comment]
ping?

**Comment by jserv** [comment]
Subject of Git commit message should not contain "`" notation though.

**Comment by ender503** [comment]
Rebasing is required.

**Comment by jserv** [comment]
@yillkid, ping?

**Pull request merged**